main.py

```diff
@@ -79,9 +79,12 @@ async def IsMemberWorking(ctx, discorduser, date = datetime.today().strftime('%d
 
     await ctx.send(content = "Done", delete_after = 0.1)
 
-@slash.slash(name="IsEveryoneHere", description="Checks if all working 'Full Time' members are in voice channel", guild_ids=guild_ids)
+@slash.slash(name="IsEveryoneHere", description = "Checks if all working 'Full Time' members are in the meeting channel", guild_ids = guild_ids)
 async def IsEveryoneHere(ctx):
 
+    today = datetime.today().date()
+    today = datetime(today.year, today.month, today.day)
```

Comment:
are you doing this to set the time to 12 am? there should be a better way to do so without reassigning a variable (which you seem to do often, try to reduce that)

Reply:
This is done to zero out the time part of the datetime object. When using datetime.today(), you get values for the time that cause an error when comparing to the dates in the database.

Reply:
Read my comment again please. 
> there should be a better way to do so

Reply:
This is an item I wanted to talk about for the upcoming sprint. Making the use of datetime/date consistent across all files.
For now would it be preferable to make a utility function to zero out the time from datetime objects?

Reply:
just checked, seems like `date` objects are slated to get a `.datetime()` function soon. this is fine for now